deployment/docker-compose.yml

```diff
@@ -19,12 +19,13 @@ services:
             JULIA_PKG_SERVER_STORAGE_SERVERS: "${STORAGE_SERVERS:-https://us-east.storage.juliahub.com,https://kr.storage.juliahub.com}"
         labels:
             com.centurylinklabs.watchtower.scope: "pkgserver.jl"
+            autoheal: "true"
         healthcheck:
-            test: curl -f --retry 5 --max-time 5 --retry-delay 10 --retry-max-time 60 "http://localhost:8000/meta" || bash -c 'kill -s 15 $$(pidof julia) && (sleep 10; kill -s 9 $$(pidof julia))'
-            interval: 2m
-            timeout: 10s
+            test: ["CMD", "curl", "-f", "-m", "1", "http://localhost:8000/meta"]
```

Comment:
1 sec is ok?

Reply:
It usually completes in <20ms for me, so I think 1s should be plenty.  In any case, it should take 3 failures in a row to cause it to get killed.